### DISCUSSION TITLE
Performance Plans - Add Ultimate Performance Profile

### DISCUSSION THREAD

**Discussion by LteClipotzel** [ChrisTitusTech/winutil#648] [Q&A]
Hello, i used the software in order to cut some of the processes on my laptop (i had constantly around 200 processes). Under Tweaks i saw this option, Add Ultimate Performance Plan, and i was wondering if its safe to use and what it does, because i couldn't find any video or explanation on your website about it.

**Comment by DeveloperDurp** [comment]
Chris made a mention of it in one of his videos, not recommended for laptops as it increases the power usage. 

https://youtube.com/watch?v=vXyMScSbhk4&si=EnSIkaIECMiOmarE&t=446

**Comment by anonyao** [comment]
> Hello, i used the software in order to cut some of the processes on my laptop (i had constantly around 200 processes). Under Tweaks i saw this option, Add Ultimate Performance Plan, and i was wondering if its safe to use and what it does, because i couldn't find any video or explanation on your website about it. 

Your shall use laptop button for laptop. And mark egde - ms.store apps button it disable process , how can u hav 200? What u even install on your pc. I have maximum 60-70 withh full windows 10 pro/ games and drivers. Disable it all.  Otherwise if u know what u doing but its your risk "privacy sexy" but it is brake settings::) but i use it
